.travis.yml

```diff
@@ -10,7 +10,7 @@ dist: xenial
 before_install: |
     set -ev
     set -o pipefail
-    npm install -g @alrra/travis-scripts
+    npm install -g @alrra/build-scripts
```

Comment:
Is this rename intentional?

Reply:
🤔  don't actually remember changing this on, i will revert the change